docs/migrating_to_v2.rst

```diff
@@ -0,0 +1,262 @@
+.. currentmodule:: discord
+
+.. _migrating_2_0:
+
+Migrating to v2.0
+=================
+
+v2.0 introduced new Discord features and depcreated some old ones.
+
+Part of the redesign involves making application commands and components. These changes include a new :class:`Bot` instance, :class:`ui.View`, and a new :class:`ApplicationContext` class. If interested in creating them, please check out our :resource:`guide <guide>`.
+
+Python Version Change
+---------------------
+
+In order to make development easier and also to allow for our dependencies to upgrade to allow usage of 3.8 or higher,
+the library had to remove support for Python versions lower than 3.7, which essentially means that **support for Python 3.7 and below
+has been dropped**.
+
+Major Model Changes
+-------------------
+
+Below are major changes that have happened in v2.0
+
+Dropped User Accounts Support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before v2.0, user accounts were supported. This is against the spirit of the library and has been removed. Thus, these features that were only applicable to them are removed:
+
+- ``bot`` argument of :meth:`Client.start` and :meth:`Client.run`
+- ``afk`` argument of :meth:`Client.change_presence`
```

Comment:
What doesn't seem related to user bots?

Reply:
Line 30

Reply:
> afk (Optional[[bool](https://docs.python.org/3/library/functions.html#bool)]) – Indicates if you are going AFK. This allows the discord client to know how to handle push notifications better for you in case you are actually idle and not lying.afk (Optional[[bool](https://docs.python.org/3/library/functions.html#bool)]) – Indicates if you are going AFK. This allows the discord client to know how to handle push notifications better for you in case you are actually idle and not lying.

From discord.py docs of 1.7.3

Reply:
Ah